zanata-war/src/main/java/org/zanata/action/DashboardAction.java

```diff
@@ -59,7 +60,8 @@
 
 @Name("dashboardAction")
 @Scope(ScopeType.PAGE)
-@Restrict("#{identity.loggedIn}")
+@ZanataSecured
+@CheckLoggedIn
```

Comment:
- [ ] Is `@ZanataSecured` needed to make `@CheckLoggedIn` work?  
- [ ] Should we put `@Interceptors(ZanataSecurityInterceptor.class)` on the `@interface CheckLoggedIn` too?
- [ ] Is there a sanity test to make sure that classes which need `@ZanataSecured` actually have it?